facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -52,7 +49,11 @@ object LiveFapiFrontPress extends FapiFrontPress {
   def pressByPathId(path: String): Future[Unit] =
     getPressedFrontForPath(path)
       .map { pressedFront => S3FrontsApi.putLiveFapiPressedJson(path, Json.stringify(Json.toJson(pressedFront)))}
-      .asFuture.map(_.fold(e => throw new RuntimeException(s"${e.cause} ${e.message}"), _ => ()))
+      .asFuture.map(_.fold(
+        e => {
```

Comment:
didn't realise it was an either (I'm just trying to illustrate my point about readability by misreading it :stuck_out_tongue: ) but `Either` way I'm happy, just in general we should be careful about overusing scala's power to do everything on one line :)